### PR TITLE
Clean up leftover dlstreamer code

### PR DESCRIPTION
**Pull request by jim-wang-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**392**

## Test Instructions if applicable

- git clone this PR
- unit test: `make unit-test-profile-launcher`
- smoke test: `make run-smoke-tests`
-

### CODE REVIEW

**Review by ejlee3 — APPROVED** [review]
LGTM

File: benchmark-scripts/README.md [hunk 1]
```diff
@@ -1,80 +1,3 @@
 # Benchmarking
```
Comment (brian-intel) [inline benchmark-scripts/README.md:1]: We can remove this entire file now that it is obsolete.

**Review by antoniomtz — APPROVED** [review]
LGTM

File: benchmark-scripts/README.md (hunk 1 above)
Comment (jim-wang-intel) [reply benchmark-scripts/README.md:1]: ok

**Review by brian-intel — APPROVED** [review]

**Review by vli11 — APPROVED** [review]
LGTM

**Pull request merged**